src/interfaces/cli.js

```diff
@@ -15,8 +15,7 @@ class CliClient {
   }
 
   sendMessage(channel, message) {
-    console.log(`[${this.nick} => ${channel}]\n${message}`);
-    console.log(`[${channel} => ${this.nick}]`);
+    console.log(message);
```

Comment:
@ananyo2012 When we use promises, there are problems with this format because this does not gets printed instantly.